docroot/themes/humsci/humsci_airy/humsci_airy.info.yml

```diff
@@ -0,0 +1,18 @@
+name: HumSci Airy
+type: theme
+base theme: humsci_basic
+description: A light and airy theme.
+package: Humsci
```

Comment:
do we want any more confirmation on this package naming from SWS?

Reply:
We just put this because we didn't want to put "Core" (since that wouldn't be accurate).

I just searched the codebase and saw that the HumSci-specific modules use `'Humanities & Sciences'` as the package name, so let's go with that precedent. I'll push an update.
![image](https://user-images.githubusercontent.com/1256329/70354332-42bd5f00-183d-11ea-833c-1ff5223e3ae6.png)